### PR TITLE
another week

### PR DESCRIPTION
**Pull request by djarecka**
@satra ping

### CODE REVIEW

File: ME-MYSELF-I/Weekly-meetings.md [hunk 1]
```diff
@@ -23,11 +23,49 @@
...
+* new API
+
+#### Where do you need help from Satra?
+
+* regtest 
+* posters
+* presentation
+
+#### What else did you do this week week?
+* helping with SIAM award application
```
Comment (satra) [inline ME-MYSELF-I/Weekly-meetings.md:62]: what's this about?

File: ME-MYSELF-I/Weekly-meetings.md [hunk 2]
```diff
@@ -23,11 +23,49 @@
...
+
+#### Who helped you this week?
+* talking to Satra about regtest (plots, specs)
+
+#### What did you achieve?
+* regtest: allowing for multiple files in tests 
+* regtest: allowing for 2d output from tests
+* regtest: proper merging of results from multiple tests
+* regtest: improving altair/vega plots
+* regtest: adding ruamel.yml to cwl_generator
```
Comment (satra) [inline ME-MYSELF-I/Weekly-meetings.md:43]: i was glad to see this.

File: ME-MYSELF-I/Weekly-meetings.md [hunk 3]
```diff
@@ -23,11 +23,49 @@
...
+* regtest: prepering some cases for the conference
+* poster for the bldg.46 retreat
+* presentation and exercises for the repronim tutorial
+* new API
+
+#### Where do you need help from Satra?
+
+* regtest 
+* posters
+* presentation
```
Comment (satra) [inline ME-MYSELF-I/Weekly-meetings.md:59]: let me know what kind of help.

**Comment by satra** [comment]
lgtm.

File: ME-MYSELF-I/Weekly-meetings.md (hunk 1 above)
Comment (djarecka) [reply ME-MYSELF-I/Weekly-meetings.md:62]: my former collaborator wanted to submit [libmpdata](https://github.com/igfuw/libmpdataxx) project to [this](http://www.siam.org/prizes/nominations/nom_wilkinson_ns.php). I was participating in the early stage of the project and I'm a co-author of the first paper.

Comment (satra) [reply ME-MYSELF-I/Weekly-meetings.md:62]: ah - thank you and good luck.

File: ME-MYSELF-I/Weekly-meetings.md (hunk 2 above)
Comment (djarecka) [reply ME-MYSELF-I/Weekly-meetings.md:43]: at the beginning I didn't realize how much I will have to change the `cwl` files and that they are just `yml` files...
I've also started testing [python-cwlgen](https://github.com/common-workflow-language/python-cwlgen), but I think for now it doesn't support everything what I'm using, so decided to give up with this.

File: ME-MYSELF-I/Weekly-meetings.md (hunk 3 above)
Comment (djarecka) [reply ME-MYSELF-I/Weekly-meetings.md:59]: i will

**Pull request merged**